test/common/utils/random.js

```diff
@@ -117,7 +117,7 @@ random.account = function() {
 random.transaction = function(offset) {
 	return lisk.transaction.transfer({
 		amount: 1,
-		passphrase: accountFixtures.genesis.password,
```

Comment:
@4miners I am confused here, issue describe to rename 

```
key -> password
```
It does not refer any change required for `passphrase`

Reply:
Now that `key` is named `password`, it's crucial to distinguish passphrases from passwords!